src/ApiService/ApiService/ServiceConfiguration.cs

```diff
@@ -26,7 +26,9 @@ public interface IServiceConfig {
 
     public string? DiagnosticsAzureBlobContainerSasUrl { get; }
     public string? DiagnosticsAzureBlobRetentionDays { get; }
-
+    public string? CliAppId { get; }
```

Comment:
Let's use the full form here (and update the config setting name)

```suggestion
    public string? ClientId { get; }
```

Reply:
I'd like to keep the naming as is. In the deploy script, we already distinguish between a `client_id` and `cli_app_id`, who's uses are distinct.